qiskit_experiments/library/randomized_benchmarking/rb_utils.py

```diff
@@ -288,6 +298,11 @@ def calculate_2q_epg(
             n_gate_2q = gates_per_clifford.get(
                 (qubit_pair, gate_2_qubit_type), 0
             ) + gates_per_clifford.get((inverse_qubit_pair, gate_2_qubit_type), 0)
-            epg = 3 / 4 * (1 - alpha_c_2q) / n_gate_2q
-            epg_2_qubit[qubit_pair] = {gate_2_qubit_type: epg}
+            epg_value = 3 / 4 * (1 - alpha_c_2q) / n_gate_2q
+            epg_stderr = epc_2_qubit_stderr / (n_gate_2q * alpha_c_1q)
```

Comment:
Why this error value is computed differently from the `epg_value`? Is this theoretical bound?

Reply:
A simplified example: if we have a random variable X and we define Y = X + 1, then sigma(Y) = sigma(X), not sigma(Y) = 1+\sigma(X).

So when computing epg_stderr we ignore constant additions as well, which simplifies things.

Reply:
ah, okey. I understood you simplified it. The std error of `alpha_c_2q` is not calculated.